apps/condo/domains/common/components/HeaderActions.tsx

```diff
@@ -110,7 +114,7 @@ export const ButtonHeaderActions: React.FC<IRightButtonHeaderActionsProps> = (pr
     const { isMobile } = useContext(AuthLayoutContext)
     const { isSmall } = useLayoutContext()
     return (
-        <Row justify={'space-between'} gutter={[20, 0]}>
+        <Row justify={'space-between'} gutter={[5, 10]}>
```

Comment:
Move gutter to constants.
`const IconContainerGutter: RowProps['gutter'] = [5, 10]`

Reply:
Really it is necessary to transfer each gutter in constants? 🤔

Reply:
Usually these values can be reused elsewhere on the same page

Reply:
This approach has been proposed by @akarjakin. This is also due to how React works with the render tree under the hood